### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by AhmedGamal-1** [community/community#110876] [GitHub Education] [answered]
### Hi everyone,

Hi, I was authenticated as a student since 1 march, but I still didn't get the student developer pack benifits can you please help me ?

![Capture](https://github.com/community/community/assets/63426707/d54fda08-89da-4827-acfe-2ad3145a5eae)
![image](https://github.com/community/community/assets/63426707/f821f5d6-617c-4fdc-bde1-3a9b47413392)

**Comment by thekarthikr** [comment, marked as answer]
It took me five days to approve. Be patient.

![image](https://github.com/community/community/assets/111447060/2e080fcc-0438-4664-a6c0-fdbba7a5538c)
